zebra-chain/src/work/equihash.rs

```diff
@@ -24,6 +31,25 @@ impl Solution {
     /// The length of the portion of the header used as input when verifying
     /// equihash solutions, in bytes
     pub const INPUT_LENGTH: usize = 4 + 32 * 3 + 4 * 2;
+
+    /// Returns true if the header is valid based on its `EquihashSolution`
+    pub fn check(&self, block: &Header) -> Result<(), Error> {
+        let n = 200;
+        let k = 9;
+        let nonce = &block.nonce;
+        let solution = &self.0;
+        let mut input = Vec::new();
+
+        block
+            .zcash_serialize(&mut input)
```

Comment:
@dconnolly this looks like a spurious "not covered by tests" annotation - there are plenty of tests that run this function.

What can we do about that?

(I'm assuming you're the person who enabled the coverage annotations?)